### ISSUE TITLE
no highlight after a successful "create"

### ISSUE THREAD

**Issue by aleray** [BigBlueHat/annotator-couchapp#1]
Hi,

I'm trying to use your code with couchdb (not pouchdb). But for some reason it does not trigger any highlight not the "annotationCreated" hook.

I suspect this is because your/couchdb api implementation does not follow [the standard annotator (2.x) API](http://docs.annotatorjs.org/en/latest/modules/storage.html#create) and returns `{"ok":true,"id":"xxx","rev":"xxx"}` instead of the document itself.

If I change annotator http storage code from:

```javascript
HttpStorage.prototype.create = function (annotation) { 
    return this._apiRequest('create', annotation);
};
```
to:

```javascript
HttpStorage.prototype.create = function (annotation) { 
    this._apiRequest('create', annotation);
    return annotation;
};
```
then it works.

What do you think? What is the best way to solve this?

Thanks

**Comment by aleray** [comment]
Someone on irc pointed me to https://wiki.apache.org/couchdb/Document_Update_Handlers in case it helps...

**Comment by BigBlueHat** [comment]
Yep! You're spot on. This is the file that needs updating to match the Annotator preferred format:
https://github.com/BigBlueHat/annotator-couchapp/blob/master/couchapp/updates/annotation.js

I'll try and take a look soon. Sorry I've been out of IRC (despite what it looked like...).
